uiplib/gui/mainGui.py

```diff
@@ -130,4 +130,8 @@ def favourite(self):
             messagebox.showinfo("Error!",
                                 "No images found. Refresh and try again later",
                                 icon='warning')
-        copy(self.images[self.index], self.settings['fav-pics-folder'])
+        image = os.listdir(self.settings['pics-folder'])[self.index]
+        image_path = os.path.join(self.settings['pics-folder'], image)
```

Comment:
you can even do `image_path = self.images[self.index].image_path` see `uipImage.py`